backend/learn/tests/test_credits.py

```diff
@@ -0,0 +1,104 @@
+from django.test import TestCase
+from learn.models import Block, Task, Level, Student, Toolbox, TaskSession
+from learn.credits import get_credits, get_earned_credits, \
+    total_credits_for_levels
+
+import pytest
+
+
+class BlockTestCase(TestCase):
```

Comment:
I think that the `BlockTestCase` is included in this module by mistake, since it doesn't test credits and it's already in `test_models.py`.